packages/amplify-category-auth/src/provider-utils/awscloudformation/utils/amplify-meta-updaters.ts

```diff
@@ -16,7 +18,7 @@ export const getPostAddAuthMetaUpdater = (context: any, resultMetadata: { servic
     providerPlugin: resultMetadata.providerName,
   };
   const parametersJSONPath = path.join(context.amplify.pathManager.getBackendDirPath(), 'auth', resourceName, 'parameters.json');
-  const authParameters = JSONUtilities.readJson<{ dependsOn: any[]; triggers: string; identityPoolName: string }>(parametersJSONPath)!;
+  const authParameters = JSONUtilities.readJson<$TSAny>(parametersJSONPath)!;
```

Comment:
Instead of using `$TSAny`, can we expand on the previous return type? Perhaps store a named `type` somewhere.